Src/AutoFoqUnitTest/FixtureIntegrationTest.fs

```diff
@@ -66,3 +67,24 @@ let FixtureSuppliesValuesToAbstractGenericTypeWithNonDefaultConstructorWithMulti
     verify <@ not <| (Unchecked.defaultof<int> = result.Property1) @>
     verify <@ not <| (Unchecked.defaultof<int> = result.Property2) @>
     // Teardown
+
+type IInterface =
+    abstract ReturnSomething : unit -> string
+
+[<Fact>]
+let ``Fixture supplies return values for non-explicitly setup test-doubles`` () =
+    let sut = (Fixture().Customize(AutoFoqCustomization())).Create<IInterface>()
+    let actual = sut.ReturnSomething()
+    verify <@ not (isNull actual) @>
+
+[<Fact>]
+let ``Fixture customizations are propagated in Foq for non-explicitly setup test-doubles`` () =
+    let fixture = Fixture()
+    let expected = "Friday, 14 December 1984"
+    fixture.Register<string>(fun () -> expected)
+    fixture.Customize(AutoFoqCustomization()) |> ignore
+    let sut = fixture.Create<IInterface>()
+
+    let actual = sut.ReturnSomething()
+
+    verify <@ expected = actual @>

```

Comment:
Fewer unit tests (deletions from `AutoFoqCustomizationTest`) and more [Facade Tests](http://blog.ploeh.dk/2012/06/27/FacadeTest): yay :+1: